### ISSUE TITLE
VERSION_STRING used to check file versions 

### ISSUE THREAD

**Issue by nathanfranklin** [simphony/AViz#10]
In fileFunctions.cpp, VERSION_STRING is used to check if files (param files etc) are of an acceptable version.  I think a version number (major, minor..) should be used instead.

**Comment by nathanfranklin** [comment]
For example, the version string that is written to file looks currently like the following:
```
"AViz AtomicVizualization version 6.1\n"
"(C) 2001 - 2003 Computational Physics Group Israel Institute of Technology\n"\
"Technion, 32000 Haifa Israel\n"\
"Geri Wagner, Adham Hashibon"
```

Which really, we are just interested in `6.1` and not the other information provided.

**Comment by nathanfranklin** [comment]
Closed by #24.  Minor and major versions are stored in the files and compared when reading files.
